src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/cli/usecases/showemails/ShowEmailContentCLIPrompt.java

```diff
@@ -0,0 +1,41 @@
+package de.dhbw.karlsruhe.students.mailflow.external.infrastructure.cli.usecases.showemails;
+
+import de.dhbw.karlsruhe.students.mailflow.core.application.auth.AuthUseCase;
+import de.dhbw.karlsruhe.students.mailflow.core.application.email.provide.ProvideEmailsUseCase;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.Email;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.exceptions.MailboxLoadingException;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.exceptions.MailboxSavingException;
+import de.dhbw.karlsruhe.students.mailflow.external.infrastructure.cli.BaseCLIPrompt;
+
+/**
+ * @author seiferla
+ */
+public class ShowEmailContentCLIPrompt extends BaseCLIPrompt {
+
+  private final Email email;
+
+  private final ProvideEmailsUseCase provideEmailsUseCase;
+
+  private final AuthUseCase authUseCase;
+
+  public ShowEmailContentCLIPrompt(
+      Email email, ProvideEmailsUseCase provideEmailsUseCase, AuthUseCase authUseCase) {
+    this.email = email;
+    this.provideEmailsUseCase = provideEmailsUseCase;
+    this.authUseCase = authUseCase;
+  }
+
+  @Override
+  public void start() {
+    try {
+      provideEmailsUseCase.markEmailAsRead(email, authUseCase.getSessionUser().email());
```

Comment:
Somehow we need to get the information of the currently needed Mailbox. I am very unsure about what the best way for retrieving the calling mailbox is. Maybe we should add it in the constructor via 

```java
  public ShowEmailContentCLIPrompt(
      Email email, ProvideEmailsUseCase provideEmailsUseCase, AuthUseCase authUseCase, MailboxType mailboxType) {
    this.email = email;
    this.provideEmailsUseCase = provideEmailsUseCase;
    this.authUseCase = authUseCase;
    this.mailboxType = mailboxType;
  }
```

so that we cann call here this:
```suggestion
      provideEmailsUseCase.markEmailAsRead(email, authUseCase.getSessionUser().email(), mailboxType);
```